### DISCUSSION TITLE
How do you use unix domain sockets?

### DISCUSSION THREAD

**Discussion by EdwarDDay** [square/okio#1261] [Q&A] [answered]
Is there a way to get a `Source` and a `Sink` for a unix domain socket?

**Comment by swankjesse** [comment, marked as answer]
Here’s a sample from OkHttp.
https://github.com/square/okhttp/blob/b94da085684eb61f543adbde54a60f28ed0c400c/samples/unixdomainsockets/src/main/java/okhttp3/unixdomainsockets/UnixDomainSocketFactory.java#L27
